src/sentry/static/sentry/app/views/organizationEventsV2/eventModalContent.tsx

```diff
@@ -166,10 +177,12 @@ const SidebarColumn = styled('div')`
   grid-column: 2 / 3;
 `;
 
+// TODO(ts): adjust types
 const SidebarBlock = styled('div')`
```

Comment:
Can you just give this something in the template `styled('div')<{withSeparator: bool}>`

Reply:
boom. done.